Makefile.am

```diff
@@ -1,4 +1,9 @@
 ACLOCAL_AMFLAGS = -I m4
 
 SUBDIRS = src man test
-dist_doc_DATA = README COPYING.txt GFDLv1.3.txt
+dist_doc_DATA = README COPYING.txt GFDLv1.3.txt ChangeLog.md
+EXTRA_DIST = generate-changelog.sh
```

Comment:
Do we really need to distribute this script in the source code tarball?
Isn't it necessary to distribute the ChangeLog.md only?
After we generate the tarball, this script won't work anymore, because it relies in the repository information.